app/views/projects/_form.html.erb

```diff
@@ -14,10 +14,19 @@
   <div class="form-group">
     <%= form.label :name, "Project name:", class: "control-label" %>
     <%= form.text_field :name, class: "form-control" %>
+    <%= form.label :description, "Project description:", class: "control-label" %>
+    <%= form.text_field :description, class: "form-control" %>
+
+    <!-- should be rewrite when creating project will be available only for logged in users -->
```

Comment:
I think we can just add `  before_action :authenticate_user!` in Projects controller - with this if user is not logged in, he will be redirected to log in page and won't be able to create new project

Reply:
Given that we're now using the filter, is the comment still relevant? Can you remove it as part of this PR @yuriy-ivanchenko ?

Reply:
@andrzejkrzywda it is already removed